static/js/containers/ProfileFormContainer.js

```diff
@@ -177,6 +113,24 @@ class ProfileFormContainer extends React.Component {
     }
   }
 
+  simpleActionHelpers: Function = (): ActionHelpers => {
+    const { dispatch } = this.props;
+    return createSimpleActionHelpers(dispatch, [
+      [setWorkDialogVisibility, "setWorkDialogVisibility"],
+      [setWorkDialogIndex, "setWorkDialogIndex"],
+      [clearProfileEdit, "clearProfileEdit"],
+      [setEducationDialogVisibility, "setEducationDialogVisibility"],
+      [setEducationDialogIndex, "setEducationDialogIndex"],
+      [setEducationDegreeLevel, "setEducationDegreeLevel"],
+      [setUserPageDialogVisibility, "setUserPageDialogVisibility"],
+      [setShowEducationDeleteDialog, "setShowEducationDeleteDialog"],
+      [setShowWorkDeleteDialog, "setShowWorkDeleteDialog"],
+      [setDeletionIndex, "setDeletionIndex"],
+      [setShowWorkDeleteAllDialog, "setShowWorkDeleteAllDialog"],
+      [setShowEducationDeleteAllDialog, "setShowEducationDeleteAllDialog"],
```

Comment:
another (cleaner?) option would be to pass in a list of the action functions an use `.name`. the util function could just use `actionCreator.name` to get the string